tools/cassandra/main.go

```diff
@@ -137,9 +137,9 @@ func buildCLIOptions() *cli.App {
 			EnvVar: "CASSANDRA_TLS_CA",
 		},
 		cli.BoolFlag{
-			Name:   schema.CLIFlagTLSEnableHostVerification,
-			Usage:  "TLS host verification",
-			EnvVar: "CASSANDRA_TLS_VERIFY_HOST",
+			Name:   schema.CLIFlagTLSDisableHostVerification,
+			Usage:  "Cassandra tls verify server hostname",
```

Comment:
[Nit] can you change the Usage wording to indicate that the flag is used to opt-out of server certificate verification?